.github/RELEASE.md

```diff
@@ -2,6 +2,15 @@
 
 # ✨ Improvements
 
+- refactor kafl command-line with subcommands (#22)
+    - move kafl scripts into a single kafl entrypoint:
+        - `kafl_fuzz.py`    -> `kafl fuzz`
+        - `kafl_debug.py`   -> `kafl debug`
+        - `kafl_cov.py`     -> `kafl cov`
+        - `kafl_plot.py`    -> `kalf plot`
+        - `kafl_gui.py`     -> `kafl gui`
+        - `scripts/mcat.py` -> `kafl mcat`
+
```

Comment:
- option --afl-skip-ranges has been deprecated (never used anyway)
- removed config override via $PWD/kafl.yaml (not explicit, easy to cause reproduction crisis)

Reply:
Thanks !